metrics.go

```diff
@@ -363,7 +363,7 @@ func labelDescriptorsFromProto(defaults map[string]labelValue, protoLabelKeys []
 }
 
 func metricProseFromProto(metric *metricspb.Metric) (name, description, unit string, ok bool) {
```

Comment:
This function name looks like a typo

Reply:
Nah, `prose` https://www.merriam-webster.com/dictionary/prose